### DISCUSSION TITLE
My wallet Firefly does not load

### DISCUSSION THREAD

**Discussion by Riponkata** [iotaledger/firefly#2170] [Help & Support] [answered]
Hello,
I sent 1034 Miota from BitPanda to my Firefly address. In BitPanda, the transaction was reported successful. But since then my wallet has been blocked and can't be loaded. When you try to start, a window appears with the following message:

App Version: 1.3.0
platform: win32
platformVersion: 10.0.17134
platformArchitecture: x64
cpuCount: 4
totalMem: 7987.8 MB
freeMem: 2780.4 MB
Error Type: Callback Error latestAddress
TypeError: Cannot read property 'balance' of undefined
    at file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:3052169
    at Array.map (<anonymous>)
    at file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:3052106
    at Array.forEach (<anonymous>)
    at file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:3052065
    at Array.forEach (<anonymous>)
    at file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:3052034
    at file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:3052197
    at m (file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:2027894)
    at Array.<anonymous> (file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:2027973)
    at i (file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:2027509)
    at Object.update (file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:2027565)
    at file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:3054263
    at onSuccess (file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:2114001)
    at Function.t.<computed>.<computed> (file:///C:/Users/%D0%9C%D0%BE%D1%8F%D1%82%20%D0%9A%D0%BE%D0%BC%D0%BF%D1%8E%D1%82%D1%8A%D1%80/AppData/Local/Programs/desktop/resources/app.asar/public/build/index.js:2:2108070)

I'm asking for help

**Comment by MiKeF05** [comment, marked as answer]
Hello, please see this issue to solve your problem.
https://github.com/iotaledger/firefly/issues/2131

**Comment by peto75** [comment]
![Снимок](https://user-images.githubusercontent.com/98058783/150228734-4d129c9d-afc6-45b9-949a-97a8d1aa05ef.PNG)

**Comment by peto75** [comment]
Здравствуйте. Кошелёк не загружается. Может подскажите как решить проблему.
Я уже удалял кошелёк  но ничего не меняется.Спасибо

**Reply by MiKeF05** [reply]
Which OS do you use?

**Reply by peto75** [reply]
Windows 10

**Reply by MiKeF05** [reply]
Before you try the following steps make sure to have the 24 words of your profiles.

1. Uninstall Firefly
2. Delete the folder Firefly in `C:\Users\[user]\AppData\Roaming\`. 
3. Install Firefly again
4. Restore your profile

You can either use your 24 words or your stronghold backup file + password. If you use a ledger then you need to use the ledger device to restore your profile.

**Reply by peto75** [reply]
Спасибо.уже  переустановить и ничего не меняется

**Reply by MiKeF05** [reply]
Have you also deleted the folder in AppData?

**Reply by peto75** [reply]
Да

**Comment by Riponkata** [comment]
I reinstalled Firefly and logged in. But an even bigger problem arose. The balance of the wallet is 0 Miota. I only have Assembly and Shimmer from staking. At first I thought Miota had been stolen from me. However, I do not see outgoing transactions. The graphics of my funds became very strange. For most of the month it shows 0. In one period of time it even reports a minus. Until two days ago, he was showing the right balance. Is it possible that my tokens were stolen? Or is it a bug? How can it be checked? I have all the addresses used where I received tokens. Can it be verified what happened to them?

![screen](https://user-images.githubusercontent.com/97985594/150424555-a7cbd9ae-4428-43d0-80c1-be054ccda0bc.png)

**Reply by MiKeF05** [reply]
This is actually an annoying bug in Firefly. You can check your addresses using the explorer at https://thetangle.org/.
I hope this will be resolved very soon.

**Comment by Riponkata** [comment]
I checked at https://thetangle.org/. These are my tokens from both addresses:

<img width="924" alt="wallet1" src="https://user-images.githubusercontent.com/97985594/150431748-fae33b8e-6b20-4caf-8d10-5db4e2d3248d.png">
<img width="917" alt="wallet2" src="https://user-images.githubusercontent.com/97985594/150431794-7b57e6e9-520c-4da4-adba-76b0c44348ac.png">

Indicates that they have been downloaded to one address. From there, several transactions follow to different addresses. It looks like a theft to me. On the other hand, my wallet does not account for these transactions.

**Comment by t00k** [comment]
Did you by any chance get contacted by someone who claimed to be from the support team, or whatever. And entered your 24 words on a website?

**Comment by Riponkata** [comment]
Yes, since I could not log in to my account, I looked where I could ask for help. Two places were mentioned in Firefly for problem reporting. Namely this site and the iota group in discord. I wrote in both places. In discord, I immediately received a call from the person in charge of Firefly. He explained that there is an official website through which the application should be synchronized.
Apparently this is the reason for the lack of tokens.
I have not used Discord so far, I installed it specifically because of this problem. I registered on this site for the same reason.
Okay, we understand what it is. Probably nothing can be done anymore.

**Comment by t00k** [comment]
We warn EVERYWHERE that;
⚠️ Reminder: don’t trust anyone asking for your seed, mnemonic phrase, passwords, or stronghold files even if they claim they’re IOTA Foundation members. Please report these scammers to one of the admins. ⚠️

📣There does not exist any browser version of the Firefly wallet or a mobile version of it. Use Firefly only on your desktop and download it from the official site here: https://firefly.iota.org/

⚡⚡⚡NEVER click on links to sites that someone sends you in private messages.!
There is no online website to sync your Firefly wallet. 
There is no online website that helps you recover funds. 
Never trust anyone approaching you in private messages in Discord⚡⚡⚡

Read more about Security during the Chrysalis migration here:
https://blog.iota.org/security-during-token-migration/
Iota Foundation employees will never message you first to offer any help or support!

There is no person in charge of Firefly that will contact you directly. There is no website to syncronize your wallet. These are scammers, taking advantage of people asking for help.

The most important thing in crypto is to NEVER share your private keys with ANYONE, and since you gave them away, they took your IOTA. So yes, these are gone, to scammers. File a police report.
